v2/brigadier/README.md

```diff
@@ -0,0 +1,98 @@
+# Brigadier: The JS/TS library for Brigade
+
+Brigadier is the library for writing [Brigade](https://brigade.sh) scripts.
+
+It provides core Brigade objects such as Events and Jobs that developers
+can use to construct expressive event-handling logic for their Brigade
+projects.
+
+Brigadier supports writing scripts in either JavaScript or TypeScript.
+
+## Installation
+
+[![NPM](https://nodei.co/npm/@brigadecore/brigadier.png)](https://www.npmjs.com/package/@brigadecore/brigadier)
+
+Normally, the brigadier dependency is declared at the top of a Brigade script
+and will be installed when the Brigade Worker starts up before script
+execution, like so:
```

Comment:
Not entirely accurate. In v1.x, brigadier is pre-installed on the worker image. In v2, a brigadier "polyfill" is pre-installed. This might be too much detail... or maybe not. I cannot decide.

Maybe we should say something like "you can install this library locally to facilitate script development and at runtime, this library (or a compatible replacement) will be pre-loaded?"

Needs a little more word-smithing.